governance/IP_POLICY_GUIDANCE.md

```diff
@@ -43,7 +43,20 @@ By using this format, the project avoids having to deal with names of copyright
 
 _Please note that you **must not** change or remove existing copyright lines unless you put them there **and** have the right to do so.
 If there are existing copyright lines add the recommended copyright statement below the existing copyright lines.
-You may however update copyright lines that say "JS Foundation" or "Node.js Foundation" to "OpenJS Foundation" as this simply reflects the merged status of the organizations._
+You may however update copyright lines that say "JS Foundation" or "Node.js Foundation" to "OpenJS Foundation" as this simply reflects the merged status of the organizations.
+The same applies for individual projects that change names._
```

Comment:
I'm not sure this is great to have in the guidance, as changing the project's name might have trademark implications that the foundation might need to look into.

Reply:
Ah, good point. The reason I added this change is because this document does not recommend (and neither did our slack convo) using "OpenJS Foundation" in the copyright. And we said that we don't think anyone transferred copyright to the foundation when on-boarding. So if you don't think we should call this out, then I think we should delete this line entirely since it is confusing.

Reply:
> And we said that we don't think anyone transferred copyright to the foundation when on-boarding.

I certainly didn't say this. The JS foundation has been around for quite a while, we can't rule out that copyright got transferred to the JS Foundation at some point.

Reply:
Do we need to confirm this more thoroughly? Maybe my wording here was incomplete saying "we said we don't think", but it for sure was said in that thread we didn't know of any. And @rginn said:

> Except in extreme cases (and this isn't an extreme case), copyrights are held by the original contributors and are not assigned to the Foundation.

Maybe Robin can help us clarify if there are any cases where copyright was transferred?

Reply:
> Do we need to confirm this more thoroughly?

I personally don't think this is high value nor worth spending resources on, but others might disagree. I also don't know whether that's the only reason to keep the copyright statement mentioning the foundation.

Reply:
Resolving via the updated PR content (I think, @tobie let me know if you disagree).